fuzzers/config_file_fuzzer.c

```diff
@@ -25,51 +26,43 @@ int foreach_cb(const git_config_entry *entry, void *payload)
 	return 0;
 }
 
-static char path[] = "/tmp/git.XXXXXX";
-static int fd = -1;
-
 int LLVMFuzzerInitialize(int *argc, char ***argv)
 {
 	UNUSED(argc);
 	UNUSED(argv);
 
 	if (git_libgit2_init() < 0)
 		abort();
-	fd = mkstemp(path);
-	if (fd < 0) {
-		abort();
-	}
 
 	return 0;
 }
 
 int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
 {
 	git_config *cfg = NULL;
+	git_config_backend *backend = NULL;
 	int err = 0;
-	size_t total = 0;
 
-	if (ftruncate(fd, 0) !=0 ) {
-		abort();
-	}
-	if (lseek(fd, 0, SEEK_SET) != 0) {
-		abort();
+	err = git_config_new(&cfg);
+	if (err != 0) {
+		goto out;
 	}
 
-	while (total < size) {
-		ssize_t written = write(fd, data, size);
-		if (written < 0 && errno != EINTR)
-			abort();
-		if (written < 0)
-			continue;
-		total += written;
+	err = git_config_backend_from_string(&backend, (const char*)data, size);
```

Comment:
Nit: there's a space missing between `const char` and `*)` and same style comment as above